### DISCUSSION TITLE
Uploads and downloads are very slow

### DISCUSSION THREAD

**Discussion by XTUxiongda** [minio/minio#16834] [Q&A] [answered]
Last night, I uploaded and downloaded files very slowly, taking up to 10 to 20 seconds
[profile.zip](https://github.com/minio/minio/files/10997041/profile.zip)
The above is the corresponding profile file, Is it caused by my incorrect usage?
Can you give me some advice?
I analyze the possible reasons by myself as follows：
1）I have some directories that are always on, and the mc mirror is synchronizing in real time
2）I have some buckets configured with replica policies and have been synchronizing to another minio cluster
The size is about 5TB

**Comment by harshavardhana** [comment, marked as answer]
Performance reviews are part of paid subscription, please join us here https://min.io/pricing
